release/arm64/build_env.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM dokken/centos-stream-9
+
+ARG env_folder
+
+COPY ${env_folder} /env_folder
+
+RUN yum update && yum install -y curl git
+
+RUN /${env_folder}/os/centos-9.sh install MEMGRAPH_BUILD_DEPS
+RUN /${env_folder}/os/centos-9.sh install TOOLCHAIN_RUN_DEPS
+
+RUN rm -rf /env_folder
+
+RUN yum clean all
+
+RUN curl https://s3.eu-west-1.amazonaws.com/deps.memgraph.io/toolchain-v4/toolchain-v4-binaries-centos-9-arm64.tar.gz -o /tmp/toolchain.tar.gz \
```

Comment:
Can you give me an example of what it would look like?
Would you then manually send the version on `docker build` or have a default value that will be used?

Reply:
toolchainc-v4Something similar exists here https://github.com/memgraph/memgraph/blob/master/release/package/ubuntu-20.04/Dockerfile, I would not provide a default, since default values will change, just pass build-arg in bash script as you do for `env_folder`. In this case that would be `toolchain-v4`

Reply:
sure, makes sense